src/components/Composer.vue

```diff
@@ -15,82 +16,165 @@
 				:placeholder="t('mail', 'Select account')"
 				:clear-on-select="false"
 				@select="onAliasChange" />
+				<button 
+					title="" 
+					:class="['icon-more',{'active':!autoLimit}]"
+					@click.prevent="autoLimit = !autoLimit">
+				</button>
+			</div>
 		</div>
 		<div class="composer-fields">
 			<label class="to-label" for="to">
 				{{ t('mail', 'To') }}
 			</label>
-			<Multiselect
-				id="to"
-				ref="toLabel"
-				v-model="selectTo"
-				:options="selectableRecipients"
-				:taggable="true"
-				label="label"
-				track-by="email"
-				:limit="4"
-				:multiple="true"
-				:placeholder="t('mail', 'Contact or email address …')"
-				:clear-on-select="true"
-				:close-on-select="false"
-				:show-no-options="false"
-				:preserve-search="true"
-				:hide-selected="true"
-				:loading="loadingIndicatorTo"
-				@input="callSaveDraft(true, getMessageData)"
-				@tag="onNewToAddr"
-				@search-change="onAutocomplete($event, 'to')" />
-			<a v-if="!showCC"
-				class="copy-toggle"
-				href="#"
-				@click.prevent="showCC = true">
-				{{ t('mail', '+ Cc/Bcc') }}
-			</a>
+			<div class="composer-fields--custom">
+				<Multiselect :class="{'opened': !autoLimit}"
+					id="to"
+					ref="toLabel"
+					v-model="selectTo"
+					:options="selectableRecipients"
+					:taggable="true"
+					label="label"
+					track-by="email"
+					:multiple="true"
+					:placeholder="t('mail', 'Contact or email address …')"
+					:clear-on-select="false"
```

Comment:
this need to be true, otherwise we need to delete the text everytime we want to add a new user

Reply:
Good afternoon! Well, in theory, yes, clear-on-select should be true. at least because it's more convenient. However, having about 500 contacts, the interface begins to noticeably blunt. it would be nice to be able to upload say 50-100 results. It seems to me that Multiselect has such an opportunity. Now I will collect all your comments and try to correct everything.